mw-webapp/src/styles/_variables.scss

```diff
@@ -15,6 +15,7 @@ $progressBar-bg: rgb(255, 255, 255);
 $font-size-large: 48px;
 
 // Colors
+$lightyellow: rgb(246, 246, 150);
 $lightblue: rgb(85, 122, 149);
 $lightgray: rgb(211, 211, 211);
 $lightgray2: #efefef;
```

Comment:
I think that better to use words primary, secondary and etc for colors vars/ Because if you will want to change design in this case you will need to rewrite all colors vars in each files but if you will use primary, secondary and etc you will need to change color only in global vars file

Reply:
We already discussed this in the chat, but in the end we stuck with this option

Reply:
Maybe I something didn't saw. You can ask again this question// Now we have designer or ask Victor...